docs/reference-guide/streams/overview.md

```diff
@@ -47,6 +47,13 @@ The `BATCH_INTERVAL` starts when the:
 - the processing of the previous batch is completed
 - the previous batch interval ended without receiving any messages
 
+The user who executes the create query are going to be owner of the stream. The
```

Comment:
Maybe highlight create query?

Reply:
What do you mean? "**create query**" or "`CREATE` query" or something else?

Reply:
More like `Create query`? I think some emphasis would be good

Reply:
`CREATE` query is I think the "correct" way